votor/src/certificate_pool/vote_pool.rs

```diff
@@ -102,8 +102,10 @@ impl SimpleVotePool {
         true
     }
 
-    pub fn add_to_certificate(&self, output: &mut VoteCertificate) {
-        output.aggregate(self.vote_entry.transactions.iter())
+    pub fn add_to_certificate(&self, output: &mut VoteCertificateBuilder) {
+        output
+            .aggregate(&self.vote_entry.transactions)
+            .expect("Incoming vote message signatures are assumed to be valid")
```

Comment:
just double checking, we've made all the appropriate checks in the vote pool such that the two errors in `aggregate` can't happen right?

https://github.com/anza-xyz/alpenglow/blob/eba6a912468773b970921929a2e4eab5c888dfdc/votor/src/certificate_pool/vote_certificate_builder.rs#L64

and 

https://github.com/anza-xyz/alpenglow/blob/eba6a912468773b970921929a2e4eab5c888dfdc/votor/src/certificate_pool/vote_certificate_builder.rs#L71

Reply:
Yes, they should not happen unless our check was screwed up, in that case we can panic.

Reply:
Yeah both of these checks should be guaranteed by the sigverify stage.
- If the sigverify stage cannot find a valid public key to verify the signature, then it will discard the vote or cert packet
- The only way the `aggregate_with` function will fail is if the signature is invalid. The sigverify stage will obviously guarantee this as well